src/utils/cache.rs

```diff
@@ -160,3 +170,15 @@ fn pipeline_delete(con: &redis::Connection, keys: Iter<String>) {
     }
     pipeline.execute(con);
 }
+
+fn execute_hset(con: &redis::Connection, hash: &str, field: &str, value: &str) {
```

Comment:
no need to write this yourself, they already exist: https://github.com/mitsuhiko/redis-rs/blob/master/src/commands.rs#L499